Code/GraphMol/ScaffoldNetwork/catch_tests.cpp

```diff
@@ -274,6 +274,7 @@ TEST_CASE("addMolToNetwork", "[unittest][scaffolds]") {
     auto m = "CC(=O)Oc1ccccc1C(=O)O aspirin"_smiles;
     REQUIRE(m);
     // check that the name was parsed into "_Name"
+    CHECK(m->hasProp("_Name"));
```

Comment:
just a comment: this is a nice-to-have, but I think it's not necessary... if the property isn't there, you'll get a KeyError on the next line and that will cause the tests to fail, which is the idea.

As written here: if the property is missing you'll get a nice catch2 error since you are using a `CHECK`, but then the next line is still going to execute and will still generate a KeyError.
If you want to avoid that KeyError, then you need to convert this `CHECK` into a `REQUIRE`

Reply:
I've taken that CHECK out but left the other asserts at CHECK. Until now I haven't really used catch2 much.